03-client-api/01-java.md

```diff
@@ -8,7 +8,25 @@ templatePath: 03-client-api/references/
 
 ## Installation
 
-#### To use this client, you need a compatible TypeDB Server running. Visit our [Compatibility Table](#version-compatibility)
+### Prerequisites
+
+To use this client, you need a compatible version of TypeDB Server running. Please see the
+[Compatibility Table](#version-compatibility) to check what version do you need, depending on the TypeDB server version
+being used.
+
+Hereinafter maven expected to be used as build system for the Java project.
+
+### Add a repository with TypeDB Java client to maven
+
+Add the code below to the `pom.xml` file in your maven project.
+
+<div class="note">
+[Important]
+Make sure to use the version number obtained in the previous section instead of the `{version}` placeholder. 
+
+Otherwise, you might have the following error: `Could not find artifact com.vaticle.typedb:typedb-client:pom:{version} 
+in repo.vaticle.com`.
+</div>
```

Comment:
In general we don't need to state the obvious like this, I think; but given we've already written it, let's just keep it terse.
```suggestion
<div class="note">
[Important]
Be sure to replace the `{version}` placeholder tag with the version of Client Java you want to install.
</div>
```